src/components/templates/Search/Filters.module.css

```diff
@@ -32,7 +32,6 @@ button.filter,
 }
 
 .filter:hover,
-.filter:focus,
```

Comment:
thought(non-blocking): it would be nice to keep a `focus` styling in general for accessibility reasons. In this case we can think of this as a bonus for later and go ahead with merging the quick fix for now. Just keep this in mind for future similar cases.

Reply:
If you already have something in mind style-wise I can quickly implement it, otherwise, I'll open an issue to track this for a future iteration.

Reply:
Issue should be fine for now